src/avg.js

```diff
@@ -1,5 +1,5 @@
 let sum = require('./sum');
-
+console.log(sum([78, 2, 0]));
```

Comment:
Remove debugging `console.log` statements before submitting things for feedback.